main.go

```diff
@@ -159,53 +163,61 @@ func before(c *cli.Context) error {
 
 func start(c *cli.Context) error {
 	names := c.Args()
+        if c.GlobalBool("oneshot") {
+		log.SetLevel(log.DebugLevel)
+		log.Info("OneShot run: " + time.Now().String())
+	        notifier.StartNotification()
+		if err := actions.Update(client, names, cleanup, noRestart); err != nil {
+			log.Println(err)
+		}
+		notifier.SendNotification()
+        } else {
+		if err := actions.CheckPrereqs(client, cleanup); err != nil {
```

Comment:
`actions.CheckPrereqs` checks for multiple watchtower instances and stops them. I think we should also check if other watchtowers are running in the `oneshot` mode, if multiple ones are running we should simply exit the current execution and not stop the other containers.

Reply:
I'm more inclined to one shot watchtowers skipping CheckPrereqs altogether.